platform/nservicebus.include.md

```diff
@@ -66,3 +66,5 @@ NServiceBus is designed to work with the rest of the Particular Service Platform
 When a message fails to be processed, even after a number of retry strategies have been attempted, NServiceBus will forward the message to an [error queue](/nservicebus/recoverability/configure-error-handling.md) for manual investigation. Messages sent to the error queue are instrumented with headers containing details about the failure including a full exception stack trace. ServiceControl picks up messages from the error queue and makes them available in [ServicePulse](/servicepulse/). Once the root cause of the failure has been found and corrected, all messages caused by the same problem can be retried at once. 
 
 Additionally, each endpoint can send [heartbeat](/monitoring/heartbeats/), [health check](/monitoring/custom-checks/), and [performance metrics](/monitoring/metrics/) through the platform for visualization in ServicePulse, making it easy to see which endpoints are offline, which are ready to scale out, and which require manual intervention.
+
+The [real-time monitoring demo](https://particular.net/real-time-monitoring) provides the ability to experience the Service Platform in action. The [Platform Sample](/platform/platform-sample-package.md) provides the ability to demonstrate the Service Platform from within any Visual Studio solution, without the need to install anything. 
```

Comment:
```suggestion
The [real-time monitoring demo](https://particular.net/real-time-monitoring) provides the ability to experience the Service Platform in action. The [Platform Sample package](/platform/platform-sample-package.md) provides the ability to demonstrate the Service Platform from within any Visual Studio solution, without the need to install anything. 
```

Since there are more than one platform samples, need to add "package" as the noun.